NuRadioMC/test/SignalProp/T01test_python_vs_cpp.py

```diff
@@ -9,6 +9,14 @@
 logger = logging.getLogger('NuRadioMC.test_raytracing')
 logger.setLevel(logging.INFO)
 
+try:
```

Comment:
can you move the try/except block to right before the numba testing and raise an error in the "except" block so that we are sure that the test fails with a useful error message when numba is not available